src/main/java/com/google/sps/data/Image.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
+
+import com.google.auto.value.AutoValue;
+
+/**
+ * Class containing information about an uploaded image.
+ */
+@AutoValue public abstract class Image {
+
+  public static Image create(long id, String blobKey, long timestamp) {
+    return new AutoValue_Image(id, blobKey, timestamp);
+  }  
+  
+  public abstract long getId();
+  public abstract String blobKey();
+  public abstract long timestamp();
```

Comment:
It's a little inconsistent, to use `getId()`, but for the other two variables use `blobKey()` and `timestamp()` without `get`. Going through the @AutoValue documentation, it seems that you should rename `getId()` to `id()`.

Reply:
Thanks for letting me know. I'll change it!